Employees/Entities/NewEmployeeInput.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Employees.Entites;
+using System;
+
+namespace Employees.Entities
+{
+    public static class NewEmployeeInput
```

Comment:
There are times when you need to use `public static`, but I don't think this is one of them. One of the problems with `public static` is that is makes things harder to unit test. In this example your class `WriteEmployeeInputtedDataToJSONFile` hard-codes a reference to `NewEmployeeInput` - it has to, because it's `public static`. That means when you write a unit test for `WriteEmployeeInputtedDataToJSONFile` you end up testing `NewEmployeeInput` too, because you can't remove the hard-coded reference. If you pass `NewEmployeeInput` into `WriteEmployeeInputtedDataToJSONFile` instead as an object, you can change it for something simpler when you test.

Here's one explanation of the problem: https://www.infoworld.com/article/3571962/how-to-unit-test-static-methods-in-csharp.html